contrib/generate_profile.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+if [ $# -ne 1 ]
+  then
+    echo "Usage: generate_profile.sh Path_Of_Foundation_Build_Directory"
+    exit 1
+fi
+fdbdir=$1
+export LD_LIBRARY_PATH=$fdbdir/lib:$LD_LIBRARY_PATH
+export FDB_CLUSTER_FILE=$fdbdir/fdb.cluster
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/fdb-%p.profraw
+$fdbdir/bin/fdbmonitor --conffile $fdbdir/sandbox/foundationdb.conf --lockfile $fdbdir/sandbox/fdbmonitor.pid &
+# This profile will be ignored
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/cli-%m.profraw
+$fdbdir/bin/fdbcli -C $fdbdir/fdb.cluster --exec 'configure new ssd single'
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/mako-build-%m.profraw
+$fdbdir/bin/mako -p 64 -t 1 --keylen 32 --vallen 16 --mode build --rows 10000  --trace  --trace_format json
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/mako-run-%m.profraw
+$fdbdir/bin/mako -p 1 -t 2 --keylen 32 --vallen 16 --mode run --rows 10000 --transaction grvg7i2gr1:48cr1:48 --seconds 60 --trace $fdbdir/sandbox/logs --trace_format json
+
+# Shutdown fdbserver to trigger profile dumping
+fdbmonitor=$(cat $fdbdir/sandbox/fdbmonitor.pid)
```

Comment:
Sorry for nitpicking, I'd suggest to call them `fdbmonitor_pid` and `fdbserver_pid`

Reply:
done